integration/kubernetes/runtimeclass_workloads/pod-shared-volume.yaml

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-shared-volume
+spec:
+  runtimeClassName: kata
+  restartPolicy: Never
+  volumes:
+  - name: shared-data
+    emptyDir: {}
+  containers:
+  - name: nginx-container
+    image: nginx:1.14
```

Comment:
Why this version? Can't we just use the latest (same comment for config below)?

Reply:
@jodh-intel fixed